test-framework/pom.xml

```diff
@@ -34,6 +34,11 @@
 
     <modules>
         <module>core</module>
+        <module>db-mariadb</module>
```

Comment:
@stianst Don't you want to introduce parent db module as a aggregated dependency module for all dbs?

Reply:
Parent module wouldn't help, but we could introduce a `db-all` module perhaps?

Reply:
You can't inherit dependencies directly from a `pom` type, what you can do is create a BOM that at least eliminates the need to set version of type, so added that.